docker/localdashboard.yaml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+    parse-dashboard:
+      image: parseplatform/parse-dashboard:4.2.0-alpha.4
+      environment:
+        - PARSE_DASHBOARD_SERVER_URL=http://pi:1337/parse
```

Comment:
Let's make this generic, "<your_parse_server_ip>/parse" ?

Reply:
I will make another version, save each parameter to a standalone environment file, so users no need to edit the yaml file, users who has less experience with docker will make fewer mistakes

for the comments I didnt reply, I will do it by your way, for those I replied, will need your confirm